AutomatedTesting/Gem/PythonTests/largeworlds/dyn_veg/EditorScripts/PhysXColliderSurfaceTagEmitter_E2E_Editor.py

```diff
@@ -176,12 +176,12 @@ def validate_behavior_context():
     collider_entity.add_component("PhysX Collider")
     helper.wait_for_condition(lambda: editor.EditorComponentAPIBus(bus.Broadcast, 'IsComponentEnabled',
                                                                    collider_entity.components[1]), 5.0)
-    hydra.get_set_test(collider_entity, 1, "Shape Configuration|Shape", 7)
-    hydra.get_set_test(collider_entity, 1, "Shape Configuration|Asset|PhysX Mesh", test_physx_mesh_asset_id)
+    hydra.get_set_test(collider_entity, 2, "Shape Configuration|Shape", 7)
```

Comment:
What does that parameter do? Might be worth giving it a name. Or in python you can put the name in when you call it e.g. get_set_test(collider_entity, whatever_that_parameter_is_called=2, ...)

Reply:
That's the index of the component it's trying to do the `get_set` too. It got me as well why that was failing. It's not great because it has to track which index it is, initially in the test when the entity is created the collider is the first component (so it used 0) and now that it as removed it and added it it's the last one, which now the index would be 2 since there are 3 components now instead of 2 (I added the static rigid body).

Adding the parameter name is a good idea, I'll give it a go.